packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -8,7 +8,7 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/Cha
 -  and management of Prometheus instances.
 +annotations:
 +  catalog.cattle.io/certified: rancher
-+  catalog.cattle.io/namespace: monitoring-system
++  catalog.cattle.io/namespace: cattle-monitoring-system
```

Comment:
needs to also update namespaceOverride in values.yaml and prometheus operator URL